src/components/POLine/POLineAgreementLines/POLineAgreementLinesContainer.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import {
+  Accordion,
+  Badge,
+  Loading,
+} from '@folio/stripes/components';
+import {
+  useShowCallout,
+} from '@folio/stripes-acq-components';
+
+import {
+  AGREEMENT_LINES,
+} from '../../Utils/resources';
+
+import { ACCORDION_ID } from '../const';
+import POLineAgreementLines from './POLineAgreementLines';
+
+const POLineAgreementLinesContainer = ({ lineId, label, mutator }) => {
+  const [agreementLines, setAgreementLines] = useState();
+  const showCallout = useShowCallout();
+
+  useEffect(() => {
+    setAgreementLines();
+
+    mutator.agreementLines.GET()
+      .then(setAgreementLines)
+      .catch(() => {
+        setAgreementLines([]);
+
+        showCallout({ messageId: 'ui-orders.relatedAgreementLines.actions.load.error', type: 'error' });
+      });
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [lineId, showCallout]);
+
+  if (!agreementLines) return <Loading />;
+
+  return (
+    <Accordion
```

Comment:
will it work if you put all presentation related code in presentation component?

Reply:
I think so.
![image](https://user-images.githubusercontent.com/43407139/85003031-550a4500-b15e-11ea-885b-34c12081a88a.png)
![image](https://user-images.githubusercontent.com/43407139/85003072-62bfca80-b15e-11ea-8a34-184360c944fe.png)

Reply:
1. what's the point of screenshots here?) they doesn't proof anything, since you don't know the moment they have been taken, so it's a matter of trust and in that case words are enough;
2. Yes, I know if just move the code it will work ) the question was about solution - should we do this? It's connected with a comment below - exporting presentation component. However, here we have also some presentation pieces in container component. Maybe that's fine, but what's the purpose of exporting view component then? IMHO, in this case hierarchy would be a bit more deeper - Accordion Container+Presentation under POline dir and some common component with the displaying list of agreement lines.

Reply:
Moved accordion to `POLineAgreementLines`